control.go

```diff
@@ -16,7 +16,7 @@ type Control struct {
 }
 
 type ControlHostInfo struct {
```

Comment:
The pointer type fields in this struct look risky to me, particularly `MessageCounter` and `CurentRemote` which appear to return the pointer to the nebula-internal values, therefore allowing modification.